server/models/Contest.js

```diff
@@ -0,0 +1,30 @@
+const mongoose = require('mongoose')
+
+const ContestSchema = new mongoose.Schema({
+    title: {
+        type: String
+    },
+
+    description: {
+        type: String,
+    },
+
+    prizeAmount: {
+        type: Number
+    },
+
+    deadline: {
+        type: Date,
+    },
+
+    created: {
+        type: Date
+    },
+
+    userId: {
```

Comment:
You may want to add a reference to the User Model here rather than storing the user's id as a string :)